trp_use_case/__openerp__.py

```diff
@@ -20,25 +20,26 @@
 ##############################################################################
 {
     "name": "Use case development",
-    "version": "1.0r45",
-    "author": "Therp BV",
+    "version": "1.0r46",
+    "author": "Therp BV - Ewout Boelsums november 2015",
```

Comment:
the author is Therp BV. If you want to be mentioned, add a readme file with https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst as template and add yourself as contributor. In that case, add other contributers from the git history as well